src/layout/FooterMarketing.js

```diff
@@ -0,0 +1,67 @@
+import { Brand, Flex, FlexItem, PageSection } from '@patternfly/react-core';
+import React from 'react';
+import logoInsights from './images/logo__redhat.svg';
+
+import './FooterMarketing.scss';
+
+const FooterTraditional = () => {
+  return (
+    <React.Fragment>
```

Comment:
Extra `return` and `React.Fragment` No need to use these as you have just one `PageSection`